### DISCUSSION TITLE
Help in validation

### DISCUSSION THREAD

**Discussion by durgeksh** [pydantic/pydantic#7688] [Question]
Hi team,
I am new to Pydantic. My requirements are as below.

1. Check data type of coulmns
2. There are 5 columns out of which, I need to validate column 1 for first 500 rows only
3. Validate the data of one model's column with other model's column (Primary and Foreign key relation)

Thank you for the wonderful library.

**Comment by Kludex** [comment]
Can you share some code? It may be easier to understand what you want.
